app/src/main/java/app/nexd/android/ui/role/RoleFragment.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("SpellCheckingInspection")
```

Comment:
i'm not sure why it is needed

Reply:
in Line 35, there is the Word "YARAFANG" used in a  debug log

Log.d("YARAFANG", "auth state $it")

Will remove the debug log and the surpress SpellChecking